backend/homework/1/logging/target/classes/logback.xml

```diff
@@ -0,0 +1,17 @@
+<configuration>
+
+    <!-- Define the appenders -->
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <file>/home/hp/IdeaProjects/StudentsManager/logs.log</file> <!-- Set the desired path for the log file -->
+        <append>true</append>
+        <encoder>
+            <pattern>%d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{35} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
```

Comment:
Create another appender to append to stdout Check [this](https://www.baeldung.com/logback#example)